src/Stryker.Core/Stryker.Core/ToolHelpers/MsBuildHelper.cs

```diff
@@ -58,7 +54,10 @@ public string GetMsBuildPath(IProcessExecutor _processExecutor)
                     }
                 }
             }
-
+            if (File.Exists(@"C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\MSBuild.exe"))
```

Comment:
Now that I see this code again. Maybe this path should just be added to the fallbackLocations list. It will result in the same behavior but cleaner code.

Reply:
Good call, Ill fix it and upload the new version